hydra-tui/src/Hydra/TUI/Options.hs

```diff
@@ -24,7 +27,7 @@ parseOptions =
 parseCardanoNodeSocket :: Parser FilePath
 parseCardanoNodeSocket =
   strOption
-    ( long "node-socket"
```

Comment:
@KtorZ If we rename this, we should also rename the command line argument of `network-magic` and do the same on the `hydra-node` to stay consistent. Personally, I like short options and only think mentioning `cardano` is important on the credentials (where it's easy to get them mixed up)